src/theme/components/link.ts

```diff
@@ -101,13 +102,16 @@ const variants = {
 const Link = {
   baseStyle: {
     ...baseLinkStyles,
-    // This is needed for custom anchor elements or link components
-    // that are passed as children to the `Link` component.
+    /** This is needed for custom anchor elements or link components
+     * that are passed as children to the `Link` component. */
     a: {
       _hover: {
         color: "ui.link.secondary",
       },
     },
+    /** The span element will handle descriptive text added to aid
+     * screen readers. */
+    span: screenreaderOnly(),
```

Comment:
This seems fine but do you think we'll get some styles where a link may need a word that is bold or italicized and so the DOM will look like `<Link><span>check this link to<b>something</b> to read more</span></Link>`. Or something similar where spans are used inside Links? It may cause unintended consequences. Maybe target it with a specific id or classname.

Reply:
Yes, to make this truely scalable, I will make some adjustments to target something unique.